man/org.freedesktop.import1.xml

```diff
@@ -0,0 +1,279 @@
+      take the machine name to export as first parameter, followed by a file descriptor (opened for writing)
+      where the tar or raw file will be written. It may either reference a file on disk or a pipe/socket. The
+      third argument specifies in which compression format to write the image. It takes one of
+      <literal>uncompressed</literal>, <literal>xz</literal>, <literal>bzip2</literal> or
+      <literal>gzip</literal>, depending on which compression scheme is required. The image written to the
+      specified file descriptor will be a tar file in case of <function>ExportTar()</function> or a raw disk
+      image in case of <function>ExportRaw()</function>. Note that currently raw disk images may not be
+      exported as tar files, and vice versa, however this restriction might be lifted eventually. The call
+      returns a transfer identifier and object path for canceling or tracking the export operation, similar
+      to <function>ImportTar()</function> or <function>ImportRaw()</function> described above.</para>
+
+      <para><function>PullTar()</function> and <function>PullRaw()</function> may be used to download, verify
+      and import a system image from a web site. They take an URL argument, that should reference a tar or
+      raw file on the <literal>http://</literal> or <literal>https://</literal> protocols, possibly
+      compressed with xz, bzip2 or gzip. The second argument is a local name for the image. It should be
+      suitable as hostname, similar to the matching argument of the <function>ImportTar()</function> and
+      <function>ImportRaw()</function> calls above. The third argument indicates the verification mode for
+      the image. It may be one of <literal>no</literal>, <literal>checksum</literal>,
+      <literal>signature</literal>. <literal>no</literal> turns of any kind of verification of the image;
+      <literal>checksum</literal> looks for a <filename>SHA256SUM</filename> file next to the downloaded
+      image, and verifies any SHA256 hash value in that file again the image; <literal>signature</literal>
+      does the same but also tries to authenticate the <filename>SHA256SUM</filename> file via
+      <citerefentry project="man-pages"><refentrytitle>gpg</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+      first. The last argument indicates whether to replace a possibly pre-existing image with the same local
+      name (if <literal>true</literal>), or whether to fail (if <literal>false</literal>). Like the import
+      and export calls above these calls return a pair of transfer identifier and object path for the ongoing
+      download.</para>
+
+      <para><function>ListTransfers()</function> returns a list of ongoing import, export or download
+      operations, as created with the six calls described above. It returns an array of structures, which
+      consist of the numeric transfer identifier, a string indicating the operation (one of
```

Comment:
Maybe format the struct fields as a list (feel free to disregard if that turns out too verbose in dockbook man output)?